### ISSUE TITLE
Boolean Attributes and Multi-word Attributes Fail

### ISSUE THREAD

**Issue by JustinWilabyPrecor** [trueadm/babel-plugin-inferno#8]
The following JSX fails:

```js
<button aria-selected="false" disabled>{i18n.t('buttonText')}</button>
```

`aria-selected` needs to be quoted in the `attrs` Object (unexpected token "-")  and the boolean attribute `disabled` yields a null `attribute.value`.

Looks like this is a Babel defect but I'd like to confirm.

**Comment by NoiSek** [comment]
I can confirm this is an issue, ran into this last week. I don't think it's a Babel defect. Inferno compiles to `{key: value}` format rather than `{"key": value}` format in its templates which causes this.

**Comment by justinwilaby** [comment]
Addressed in #9

**Comment by trueadm** [comment]
@justinwilaby I do believe JSX doesn't support using hyphen in prop names, although I may be wrong here. I'll look into it a bit further.
